core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/UseKeysClause.kt

```diff
@@ -0,0 +1,44 @@
+package ch.ergon.dope.resolvable.clause.model
+
+import ch.ergon.dope.DopeQuery
+import ch.ergon.dope.resolvable.clause.IDeleteClause
+import ch.ergon.dope.resolvable.clause.IDeleteUseKeysClause
+import ch.ergon.dope.resolvable.clause.ISelectFromClause
+import ch.ergon.dope.resolvable.clause.ISelectUseKeysClause
+import ch.ergon.dope.resolvable.expression.TypeExpression
+import ch.ergon.dope.validtype.ArrayType
+import ch.ergon.dope.validtype.StringType
+
+private const val USE_KEYS = "USE KEYS"
+
+sealed class SelectUseKeysClause(
+    private val parentDopeQuery: DopeQuery,
+    private val keysDopeQuery: DopeQuery,
+) : ISelectUseKeysClause {
+    override fun toDopeQuery() = DopeQuery(
+        queryString = "${parentDopeQuery.queryString} $USE_KEYS ${keysDopeQuery.queryString}",
```

Comment:
Use the `formatToQueryStringWithSymbol` function
```
formatToQueryStringWithSymbol(parentDopeQuery.queryString, USE_KEYS, keysDopeQuery.queryString)
```

Reply:
done